ansible/roles/dev-desktop/tasks/main.yml

```diff
@@ -0,0 +1,88 @@
+---
+
+- name: remove preinstalled things
+  apt:
+    name:
+      - python3-cryptography
+    state: absent
+
+- name: install deps
+  apt:
+    name:
+      - pkg-config
+      - build-essential
+      - libssl-dev
+      - python3-pip
+
+    state: present
+
+- name: install python libs
+  shell: "pip3 install PyGithub requests cryptography"
+
+- name: install rustup in userspace for root
```

Comment:
in the current instance build, `rustup` doesn't look to be working, even though it appears to be installed here.

Reply:
rustup needs to be installed per user. There is an open issue to support multi-user environments: https://github.com/rust-lang/rustup/issues/313